src/lib/alarms/trade_offer.ts

```diff
@@ -6,6 +6,24 @@ import {AnnotateOffer} from '../bridge/handlers/annotate_offer';
 import {PingCancelTrade} from '../bridge/handlers/ping_cancel_trade';
 import {CancelTradeOffer} from '../bridge/handlers/cancel_trade_offer';
 import {FetchSteamUser} from '../bridge/handlers/fetch_steam_user';
+import {rgDescription} from '../types/steam';
+import {HasPermissions} from '../bridge/handlers/has_permissions';
+
+export interface ExtendedOfferStatus {
```

Comment:
Why not use `TradeOffersAPIOffer` instead of a near-duplicate interface? It seems like this parses `accountid_other` into a SteamID64, but that could just be handled by the caller if they want that format.